Issue1〜3の修正

koharayuki opened the pull request:
Issue1〜3の解消

koharayuki committed:
コード整理の準備

<ターミナル操作>
・bundle install(Gemfile.lock)
・yarn install(yarn.lock)

koharayuki committed:
Issue1

#37
:〇〇 =>の表記を〇〇:に変更

(month,date,plans)

koharayuki committed:
Issue2

・getweekをget_weekに変更
#5,#21

・wdaysをweek_daysに変更

※Rubyの命名規則遵守のため

koharayuki committed:
Issue3

divおよび/divの削除

不要であるため実施

TakashiKozuma commented on app/controllers/calendars_controller.rb:
```diff
@@ -18,8 +18,8 @@ def plan_params
 
-  def getWeek
-    wdays = ['(日)','(月)','(火)','(水)','(木)','(金)','(土)']
+  def get_Week
```
```rb
def get_Week
```
5,21行目はメソッド名ですので「頭文字大文字のキャメルケース」から「小文字のスネークケース」に変更する必要があります。
現状、キャメルケースとスネークケースが混ざった書き方になってしまっていますので、
改めてカリキュラムを見直して「小文字のスネークケース」に修正しましょう！

koharayuki replied:
早速のレビューありがとうございます。
再度、レビュー依頼とさせていただきましたのでご確認お願い致します。

TakashiKozuma commented on app/controllers/calendars_controller.rb:
```diff
@@ -18,8 +18,8 @@ def plan_params
-  def getWeek
-    wdays = ['(日)','(月)','(火)','(水)','(木)','(金)','(土)']
+  def get_Week
+    week_days = ['(日)','(月)','(火)','(水)','(木)','(金)','(土)']
```
```rb
week_days
```
こちらの変数名は変更する必要がないので、元に戻しておきましょう。

koharayuki replied:
本件、承知しました。
wdaysは一般的に使用する単語なのでしょうか？

TakashiKozuma reviewed (commented):
レビュー依頼ありがとうございます！
コメントを致しましたので、ご確認お願いいたします。

修正後はコミット・プッシュのし忘れがないかご確認の上、再度フォームからレビュー依頼をお願い致します。

koharayuki committed:
コメント内容の修正

・「頭文字大文字のキャメルケース」から「小文字のスネークケース」に変更

・変更する必要がない変数名を元に戻した
→wdaysは一般的に使用する単語なのでしょうか？

TakashiKozuma commented:
レビュー依頼ありがとうございます。
LGTMです！issueの解決お疲れさまでした。

次のカリキュラムにお進みください！！
